ProjectSettings/Packages/com.unity.learn.iet-framework/Settings.json

```diff
@@ -9,8 +9,8 @@
             {
                 "type": "System.Boolean, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089",
                 "key": "IET.DisplayWelcomeDialogOnStartup",
-                "value": "{\"m_Value\":true}"
+                "value": "{\"m_Value\":false}"
```

Comment:
This should be reverted, so the tutorial popup is displayed on project open.
I believe when it's displayed, this field is being set back to false.